compiler/circle-quantizer/src/CircleQuantizer.cpp

```diff
@@ -180,7 +180,16 @@ int entry(int argc, char **argv)
 
   // Load model from the file
   foder::FileLoader file_loader{input_path};
-  std::vector<char> model_data = file_loader.load();
+  std::vector<char> model_data;
+  try
+  {
+    model_data = file_loader.load();
+  }
+  catch (const std::runtime_error &err)
+  {
+    std::cerr << "circle-quantizer: ERROR: " << err.what() << std::endl;
+    return 255;
+  }
```

Comment:
Let's catch exception in app itself as much as possible to customize error message.